server/lib/budget.js

```diff
@@ -134,12 +134,18 @@ export async function getTotalNetAmountReceivedAmount(collective, { startDate, e
   return { ...totalReceived, value: totalReceived.value + totalFees.value };
 }
 
-export async function getTotalMoneyManagedAmount(host, { startDate, endDate, currency, version } = {}) {
+export async function getTotalMoneyManagedAmount(host, { startDate, endDate, collectiveIds, currency, version } = {}) {
   version = version || host.settings?.budget?.version || 'v1';
   currency = currency || host.currency;
 
-  const hostedCollectives = await host.getHostedCollectives();
-  const ids = hostedCollectives.map(c => c.id);
+  let hostedCollectives, ids;
+  if (collectiveIds?.length > 0) {
+    ids = collectiveIds;
```

Comment:
No validation of the ids here. On purpose?

Reply:
Since these `id`s are used to get the sum of all transactions at https://github.com/opencollective/opencollective-api/blob/46013b0d7e33baf7640c98a5ab4fb4b515287d68/server/lib/budget.js#L189

If an `id` cannot be found/filtered on then we just have to ignore it in my understanding and just filter on the remaining `id`s that are passed. This is the reason for not having any validation here. But we can put a validation just to print out an info log but not sure if that's really needed. 🤔